Frontend/android-native/app/src/driver/nammaYatriPartner/assets/juspay/v1-configuration.js

```diff
@@ -230,5 +230,12 @@ window.getMerchantConfig = function () {
         "languageKey" : "TA_IN"
       }
   ]
+  , "liveStatsDashboard" : {
+        "show" : true,
```

Comment:
update the keys for dashboard as used in the [PR](https://github.com/nammayatri/nammayatri/pull/4499)